### ISSUE TITLE
Update park data

### ISSUE THREAD

**Issue by luqmaan** [open-austin/austingreenmap#50]
We haven't updated the park data since Hack For Change.

To update the park data run the download.sh script. The URLs may have changed and this script may break.

Also, we should consider using the non-public parks that @wilsaj has added to the Austin Parks Equity project. https://github.com/open-austin/austin-park-equity/search?utf8=%E2%9C%93&q=osmParksLayer

**Comment by wilsaj** [comment]
Maybe this is a crazy pants idea, but what about using OSM as a basis for both projects? If we import park data into OSM, we can curate and normalize things like amenities, ownership (public/private), and hours so we don't have to do that normalization in data processing scripts or app code.

Reasons to do this:
- would simplify the data processing steps for both projects
- make these projects more forkable; all cities are OSM cities
- OSM will have high quality park data that can benefit other projects
- OSM mapping is easy to plug people into and makes for a good short-term project for a hackathon or hack night

Reasons not to:
- restrictions on source data (not public domain or compatible with OSM's ODBL license)
- a significant portion of the required data isn't appropriate for inclusion into OSM (e.g. budgeting info). We can associate OSM data with other data, but if it's not worth it then it's not worth it.

**Comment by wilsaj** [comment]
Maybe this should be opened as a separate issue. Also ping @mateoclarke

**Comment by luqmaan** [comment]
:+1: :+1: :+1: 

Importing the park data into OSM seems like a really good idea/the best use of our time (as oppose to creating a very intricate map viewer).

I think we'd need to get PARD involved:
- How is the data licensed? Does it also fall under the data portal public domain thing?
- How often do they update the data?

**Comment by mateoclarke** [comment]
Yeah I like that idea Mr. Crazy Pants! I have some contacts at PARD. Can I take these questions to them and then we can open some issues for what to do next?

Having both theses projects, and any other future Austin park project, pull from an open source database seems like a worthwhile project in itself. And it can be a direct example of why the OSM IMPORT is so important. 

So much "synergzm"!!!

🌳

**Comment by mateoclarke** [comment]
[City of Austin Open Data Terms of Use Policy](https://data.austintexas.gov/dataset/City-of-Austin-Open-Data-Terms-of-Use-Policy/2z87-8uh7)

Full list of GIS data on CoA [ArcGIS REST Services Directory](http://services.arcgis.com/0L95CJ0VTaxqcmED/ArcGIS/rest/services/)

**Comment by mateoclarke** [comment]
Email sent to Matt E asking if ArcGIS data falls under CoA Open Data Terms of Use.

**Comment by luqmaan** [comment]
Just sent a reminder email.

**Comment by luqmaan** [comment]
Matt Esquibel responded:

> Yes, the GIS Committee approved and consider the data open and unrestricted. It should fall under the general Terms of Use.  See: http://cityofaustin.github.io/open-data-manual/portal-terms-of-use/  As Hailey notes, there is still some work to be done to finalize our Terms of Use – let us know if there is something specific needed to address this data in particular.

**Comment by mateoclarke** [comment]
Sorry, should've pointed you at this a while back:
https://github.com/open-austin/liberate-the-data/issues/3
